libretro_core_options.h

```diff
@@ -55,7 +55,7 @@ struct retro_core_option_definition option_defs_us[] = {
    {
       BEETLE_OPT(renderer),
       "Renderer (Restart)",
-      "Choose video renderer. 'Software' is the most accurate but has the highest performance requirements when running at increased internal GPU resolutions. 'Hardware' selects automatically the 'OpenGL' or 'Vulkan' renderer, depending upon the current RetroArch video driver setting. While less accurate, these renderers improve performance and enable various enhancements such as texture filtering and perspective correction.",
+      "Choose video renderer. 'Software' is the most accurate but has the highest performance requirements when running at increased internal GPU resolutions. 'Hardware' selects automatically the 'OpenGL' or 'Vulkan' renderer, depending upon the current RetroArch video driver setting. While less accurate, these renderers improve performance and enable various enhancements such as texture filtering and perspective correction. Core will automatically fall back to the software renderer if 'Hardware' is selected but the RetroArch video driver is not set to Vulkan or an OpenGL 3.3-compatible driver.",
```

Comment:
Remember, libretro cores aren't exclusive to RetroArch. Consider changing this sublabel to something that is generic for any libretro frontend.

Reply:
That's true; I was basing it off the original text that already mentioned RetroArch. I'll change it to something more neutral.